### DISCUSSION TITLE
Sobre la sincronización de cotalker desde via app mediante datos moviles

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#346] [2. Questions and Answers] [answered]
Hola 

Quisiera saber si cada vez que inicio sesión en Cotalker se sincroniza todo de nuevo o deja algo ya guardado. Me parece que antes de cotalker universal (cotalker android/ios), se limpiaba el caché cuando el usuario se deslogueaba.

Entre ayer probé la app y siempre me pedía sincronización y no se detenía, hoy realicé un video desde la app, para saber si el comportamiento es el correcto o es un bug,

Agregando, en el video realicé 2 inicio de sesiones y solo la segunda me apareció las animaciones para los grupos.

Dejo el link del video: https://drive.google.com/file/d/14_L_-KslV8tvKuyervuk0hyE1v2GqdKI/view?usp=sharing

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG cada vez que inicias sesión la sincronización parte de cero, esto era consistente con el cliente antiguo.

Es un comportamiento correcto lo que muestras, en compañías con muchos datos la sincronización inicial podría demorarse, sin embargo, no te impide utilizar la aplicación mientras se sincroniza.
